Source/DataSources/TimeIntervalCollectionPositionProperty.js

```diff
@@ -57,6 +57,7 @@ Object.defineProperties(TimeIntervalCollectionPositionProperty.prototype, {
    * Gets the interval collection.
    * @memberof TimeIntervalCollectionPositionProperty.prototype
    * @type {TimeIntervalCollection}
+   * @readonly
```

Comment:
`@readonly` applies to `intervals` as well, correct?

Reply:
You might have been looking at the next field, `referenceFrame` -- yes, that should also be `@readonly`, should be fixed in the changes I'm about to push.